src/content/basics/giant-swarm-platform-access/index.md

```diff
@@ -0,0 +1,71 @@
++++
+title = "The Giant Swarm Platform Access Routes"
+description = "General Giant Swarm Platform Access Routes Overview"
+date = "2019-05-08"
+weight = 20
+type = "page"
+categories = ["basics"]
++++
+
+# Giant Swarm Platform Access Routes
+
+## Intro
+Access to Giant Swarm platforms can be split into two parts. 
```

Comment:
I find the `platforms` term not specific enough. This should be specified and aligned with the title. See my other comment.